src/output.h

```diff
@@ -413,6 +413,27 @@ std::pair<std::string, nc_color> const& get_item_hp_bar(int dmg);
 
 std::pair<std::string, nc_color> const& get_light_level(const float light);
 
+/**
+ * @return String containing the bar. Example: "Label [********    ]".
+ * @param val Value to display. Can be unclipped.
+ * @param width Width of the entire string.
+ * @param label Label before the bar. Can be empty.
+ * @param begin Iterator over pairs <double percentage, char char>.
+ * @param end Iterator over pairs <double percentage, char char>.
```

Comment:
Would be nice to explain what these iterators do. The meaning of `value`, `width` and `label` are explained, but for `begin` and `end`, it just says *what* type it expects, not what it actually does.